File: demo/src/main/java/com/baozi/demo/activity/TBActivity.java
package com.baozi.demo.activity;

import android.os.Bundle;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import com.google.android.material.tabs.TabLayout;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentPagerAdapter;
import androidx.viewpager.widget.ViewPager;
import androidx.appcompat.app.AppCompatActivity;
import com.baozi.demo.R;
import com.baozi.demo.fragment.MineFg;

public class TBActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) throws NullPointerException{
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_tb_home);
        ViewPager content = findViewById(R.id.content);
        content.setAdapter(new FragmentPagerAdapter(this.getSupportFragmentManager()) {
            @Override
            public Fragment getItem(int i) {
                return new MineFg();
            }

            @Override
            public int getCount() {
                return 5;
            }

            @NonNull
            @Override
            public CharSequence getPageTitle(int position) {
                return position + "";
            }
        });
        TabLayout tb_title = findViewById(R.id.tb_title);
        tb_title.setupWithViewPager(content);
    }
}

File: demo/src/main/java/com/baozi/demo/activity/TBActivity.kt
package com.baozi.demo.activity

import android.os.Bundle
import android.support.design.widget.TabLayout
import android.support.v4.app.Fragment
import android.support.v4.app.FragmentPagerAdapter
import android.support.v4.view.ViewPager
import android.support.v7.app.AppCompatActivity
import com.baozi.demo.R
import com.baozi.demo.fragment.MineFg

class TBActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_tb_home)
        val content = findViewById<ViewPager>(R.id.content)
        content.adapter = object : FragmentPagerAdapter(this.supportFragmentManager) {
            override fun getItem(i: Int): Fragment {
                return MineFg()
            }

            override fun getCount(): Int {
                return 5
            }

            override fun getPageTitle(position: Int): CharSequence {
                return position.toString() + ""
            }
        }
        val tb_title = findViewById<TabLayout>(R.id.tb_title)
        tb_title.setupWithViewPager(content)
    }
}
